gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Connection.java

```diff
@@ -59,14 +59,15 @@ final class Connection {
 
     public static final int MAX_IN_PROCESS = 4;
     public static final int MIN_IN_PROCESS = 1;
-    public static final int MAX_WAIT_FOR_CONNECTION = 16000;
+    public static final int MAX_WAIT_FOR_CONNECTION = 25000;
```

Comment:
Changing default could be a breaking change for users. As an example, a user may be relying on connection setup to be at 15s based on which they may have configured throttling rules in their application. 

I would suggest to target the change in default values for v3.7.x

Reply:
This seems like more of a behavioral change than a complete breaking change. However, if you think this is still too big of a behavioral change then I will remove it and we can look to change it in 3.7 as you suggested.

Reply:
You are right, it's not an interface / contract change but if we release this as a patch change, a customer may have to change their application code to consume the patch version. 

As an example, let's say that the customer has set exponential backoff/retry for connection setup based on the current default values but now we are changing the default values and their tuning for retries would break. That is the reason, I would like to add this to 3.7.x.

@spmallette thoughts?

Reply:
I'm fine with it waiting for 3.7.x.